app/services/atmosphere/optimizer.rb

```diff
@@ -56,6 +61,7 @@ def satisfy_appliance(appliance)
         end
       end
 
+      action.log("Satisfying appliance finished")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.